### PR TITLE
#1356 Grey out items with no changes

### PR DESCRIPTION
**Pull request by steveswinsburg**
Add internationalisation to the selection step page
Fix placeholders on other pages

WIP.

### CODE REVIEW

File: gradebookng/tool/src/webapp/styles/gradebook-importexport.css [hunk 1]
```diff
@@ -36,4 +36,8 @@
 #gradebookImportExport .gb-import-export-section p,
 #gradebookImportExport .gb-import-export-section form {
   margin: 1em 0 0;
+}
+
+.no_changes {
+	color: grey;
```
Comment (steveswinsburg) [inline gradebookng/tool/src/webapp/styles/gradebook-importexport.css]: feel free to change this. i went with a boring grey. See screenie:
screen shot 2016-05-31 at 11 48 10 pm

**Comment by payten** [comment]
Looks good to me! I noticed "WIP"? Am I ok to merge?

**Comment by steveswinsburg** [comment]
👍 I am doing PRs for each of the items as there are quite a few on the import/export list 🐪

**Comment by steveswinsburg** [comment]
Merging to make way for next one :)

**Pull request merged**